Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -332,6 +332,15 @@ def SwapKeepAndDrop(l):
 RAWSIMEventContent.outputCommands.extend(IOMCRAW.outputCommands)
 RAWSIMEventContent.outputCommands.extend(CommonEventContent.outputCommands)
 #
+# Temporary collections needed for Phase-2 RECO using RAWSIM as input in Prod-like workflow
+# They are until packer/unpackers are done.
+#
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(RAWSIMEventContent,
```

Comment:
I would like to discuss on this, if it should be placed here. So we can follow up easily. Or it should go under the hood of subsystem eventcontent.